3ds Max/Max2Babylon/Exporter/BabylonExporter.cs

```diff
@@ -36,33 +38,35 @@ internal partial class BabylonExporter
         public static string exporterVersion = "1.4.2";
         public float scaleFactor = 1.0f;
 
-        void ReportProgressChanged(int progress)
+        public const int MaxSceneTicksPerSecond = 4800; //https://knowledge.autodesk.com/search-result/caas/CloudHelp/cloudhelp/2016/ENU/MAXScript-Help/files/GUID-141213A1-B5A8-457B-8838-E602022C8798-htm.html
```

Comment:
Right now, i put this constant here, but it may be prudent to do another minor refactor to move some of the magic numbers floating around the codebase to a more centralized place with links to documentation for clarity